### ISSUE TITLE
Some content pushed down when intro is too short

### ISSUE THREAD

**Issue by BackEndTea** [symfony/symfony-docs#10133]
When a page has a really short intro before the code sample it only pushed part of it down, making it look a bit wonky

![image](https://user-images.githubusercontent.com/14289961/43449123-f29b8c7a-94af-11e8-8519-7baaa1f1688f.png)
https://symfony.com/doc/current/frontend/encore/vuejs.html

When there are two lines it looks a bit better, but it still collides with the box.
![image](https://user-images.githubusercontent.com/14289961/43449215-29f8b56c-94b0-11e8-92a2-805b175e7b90.png)
https://symfony.com/doc/current/frontend/encore/reactjs.html

Found on Chrome:   Version 67.0.3396.99 (Official Build) (64-bit) on Mac OS 10.13.6

**Comment by javiereguiluz** [comment]
Thanks for reporting this issue! We recently added a similar issue for note/tip/caution. See for example this page: https://symfony.com/doc/3.3/reference/configuration/assetic.html

We applied the same solution here: make the console window aware of the version selector and reduce its size if needed:

![vue](https://user-images.githubusercontent.com/73419/43904723-b093ce18-9bef-11e8-8b8c-3147c9329be7.png)
![react](https://user-images.githubusercontent.com/73419/43904722-b08e857a-9bef-11e8-9c55-afc6bc992b11.png)

The fix will be on symfony.com after the next deploy, so I'm closing it as solved. Thanks!
